packages/vscode-extension/src/project/project.ts

```diff
@@ -265,9 +265,15 @@ export class Project implements Disposable, MetroDelegate, ProjectInterface {
           this.updateProjectState({ status: "refreshing" });
           break;
         case "RNIDE_fastRefreshComplete":
-          if (this.projectState.status === "starting") return;
-          if (this.projectState.status === "incrementalBundleError") return;
-          if (this.projectState.status === "runtimeError") return;
+          if (this.projectState.status === "starting") {
+            return;
+          }
+          if (this.projectState.status === "incrementalBundleError") {
+            return;
+          }
+          if (this.projectState.status === "runtimeError") {
+            return;
+          }
```

Comment:
Please revert changes in project.ts, it's not strictly needed in this PR and will conflict with my other changes.

Reply:
@jakub-gonet changes are only caused by formatting, previously someone did not format it and the rule `Expected { after 'if' condition.eslint[curly](https://eslint.org/docs/rules/curly)` was not satisfied